README.rst

```diff
@@ -1279,6 +1279,30 @@ finds one that no longer finds a ``redirect`` directive.  At that point it will
 ``buildrunner.yaml``.  It will **not** merge entries from earlier, redirecting ``buildrunner.yaml``
 files - all other content is ignored when the ``redirect`` directive is found.
 
+Cleaning Cache
+==============
+
+Buildrunner keeps a local cache in the ``~/.buildrunner/caches`` directory that will grow over time and
+should be cleaned out periodically. There are two methods for cleaning this cache.
+
+clean-cache parameter
+---------------------
+
+You can pass in the ``--clean-cache`` parameter with any execution of ``buildrunner``, and the cache
+will be cleaned out prior to the build.
+
+.. code:: bash
+
+  buildrunner --clean-cache
+
+buildrunner_cleanup command
+---------------------------
+
+There is a stand-alone command that just cleans up the cache. This command takes no parameters.
+
+.. code:: bash
+
+  buildrunner_cleanup
```

Comment:
Personal preference for me is dashes over underscores, but that's just because I have to hit an extra key to type an underscore and I don't like doing that. :)

Reply:
But you can keep it as is, just stating my personal preference. Here's a breakdown of scripts in a bin dir with each:
https://unix.stackexchange.com/questions/49008/for-filenames-are-underscores-or-dashes-more-common-in-unix-file-names